projects/python-client/src/datapane/resources/local_report/ipython_template.html

```diff
@@ -0,0 +1,56 @@
+<style>
+  #dp-{{ report_id }} {
+    width: 1px;
+    min-width: 100%;
+  }
+</style>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/iframe-resizer/4.3.2/iframeResizer.min.js" integrity="sha512-dnvR4Aebv5bAtJxDunq3eE8puKAJrY9GBJYl9GC6lTOEC76s1dbDfJFcL9GyzpaDW4vlI/UjR8sKbc1j6Ynx6w==" crossorigin="anonymous" referrerpolicy="no-referrer"></script>
+<iframe id="dp-{{ report_id }}" srcdoc="{% filter escape %}<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <script src="https://kit.fontawesome.com/256c714d9a.js"></script>
+    <link
+      href="https://fonts.googleapis.com/css?family=Inter:400,500,600,700|Inconsolata|Karla|Rubik&display=swap"
+      rel="stylesheet"
+    />
+    <script>
+      window.dpLocal = true;
+      {% if events %}
+        window.dpLocalViewEvent = {{events|tojson}};
+        window.dpReportId = "{{report_id}}";
+        window.dpAuthorId = "{{author_id}}";
+      {% endif %}
+    </script>
+    {% if standalone %}
```

Comment:
is the `standalone` boolean used when viewing via Jupyter? Do we want to hardcode `standalone` as False instead? i.e. we always require network access?